commands/create_kubeconfig.go

```diff
@@ -44,153 +89,205 @@ func init() {
 	CreateCommand.AddCommand(CreateKubeconfigCommand)
 }
 
-// Pre-check before creating a new kubeconfig
-func checkCreateKubeconfig(cmd *cobra.Command, args []string) error {
-	kubectlOkay := util.CheckKubectl()
-	if !kubectlOkay {
-		// kubectl not installed
-		errorMessage := color.RedString("kubectl does not appear to be installed") + "\n"
-		if runtime.GOOS == "darwin" {
-			errorMessage += "Please install via 'brew install kubernetes-cli' or visit\n"
-			errorMessage += fmt.Sprintf("%s for information on how to install kubectl", kubectlInstallURL)
-		} else if runtime.GOOS == "linux" {
-			errorMessage += fmt.Sprintf("Please visit %s for information on how to install kubectl", kubectlInstallURL)
-		} else if runtime.GOOS == "windows" {
-			errorMessage += fmt.Sprintf("Please visit %s to download a recent kubectl binary.", kubectlWindowsInstallURL)
+// createKubeconfigPreRunOutput shows our pre-check results
+func createKubeconfigPreRunOutput(cmd *cobra.Command, cmdLineArgs []string) {
+	args := defaultCreateKubeconfigArguments()
+	err := verifyCreateKubeconfigPreconditions(args, cmdLineArgs)
+
+	if err != nil {
```

Comment:
something like:
```
if err == nil {
   return
}
```
would be cleaner here

Reply:
Yeah, done here and in the example.